README.md

```diff
@@ -195,6 +195,57 @@ buildctl build \
 
 `--local` exposes local source files from client to the builder. `context` and `dockerfile` are the names Dockerfile frontend looks for build context and Dockerfile location.
 
+##### Hint: `dockerfile.v0` frontend with `buildctl build --opt` option
+For `dockerfile.v0` frontend there are some parameters that can be used with `--opt`.
+The following parameters are the most frequently used.
+* target: set the target build stage to build.
+  * Example: `target=Stage1`
+* filename: name of the Dockerfile (Default is 'Dockerfile'),set to use a differently-named Dockerfile.
+  * Example: `filename=dockerfile.prod`,`filename=other.dockerfile`
+* dockerfilekey: dir path of the custom named context for Dockerfile. Useful when there is more than one build source.
+  * Full command example: 
+    * `buildctl --frontend=dockerfile.v0 --local dockerfile=. --local dockerfile2=/tmp dockerfilekey=dockerfile2` buildctl will try to use the `/tmp/Dockerfile` as Dockerfile.
```

Comment:
Is it supposed to be `--opt dockerfilekey=dockerfile2`?

Reply:
yes

Reply:
Thanks,my fault,it's now fixed.